### DISCUSSION TITLE
About Traefik Network Name

### DISCUSSION THREAD

**Discussion by YnievesDotNet** [Tecnativa/doodba#518] [Doodba - The template]
Hello Doodba's

I are putting in production Doodba. But my traefik network is not naming inverseproxy_shared, hi is called proxy?

is possible ask the network name in the copier process

Best Regards.

**Comment by YnievesDotNet** [comment]
I can try install base addons, and the run is stuck

```
user@localhost:~/apps/doodba$ docker-compose run  --rm odoo -i base --stop-after-init
Creating doodba_odoo_run ... done
doodba INFO: Waiting until postgres is listening at db...
```

Thanks

**Comment by rpsjr** [comment]
Did u soulved ur problem? 

If yes, could provide the steps to achive this? 
If not, could u provide detailed steps to reproduce this issue?
